app/scripts/controllers/detect-tokens.test.js

```diff
@@ -16,25 +15,30 @@ import { toChecksumHexAddress } from '../../../shared/modules/hexstring-utils';
 import DetectTokensController from './detect-tokens';
 import PreferencesController from './preferences';
 
-function buildMessenger() {
-  return new ControllerMessenger().getRestricted({
-    name: 'DetectTokensController',
-    allowedEvents: ['NetworkController:stateChange'],
-  });
-}
```

Comment:
This has been substituted with `getRestrictedMessenger` which uses the `messenger` instance available in the higher scope, as it helps registering a mock handler for the `KeyringController:getState` action.